lib/localize.js

```diff
@@ -20,6 +20,7 @@ const characterMap = new Map([
 	['?', 'questionMark'],
 	['"', 'quotationMark']
 ]);
+const commonTerms = ['navigation:back:title'];
```

Comment:
I grouped this under "navigation" but happy to discuss other ways to group things!

Reply:
This is great. I've noticed a lot of repeated terms elsewhere for workflow buttons (like `Cancel`, `Save`). I'm guessing they would be something like `button.save`, `button.cancel`? Not sure if it should be plural or not.

Reply:
I don't know if I'd want to tie "button" to it -- which implies the component to use -- but maybe something like "action" or "verb"? That way "Save", "Cancel", "Save & New", "Delete", "Copy" could all kind of fit in there. 🤷

Reply:
But then you could argue that "Back" is also an action/verb, so maybe not having a category at all is better?

Reply:
Sounds good to me.